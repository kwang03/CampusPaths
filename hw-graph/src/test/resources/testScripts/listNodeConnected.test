#Tests the functionality of listing nodes in a connected graph

#Creates graph
CreateGraph myGraph

#Adds some nodes to the graph
AddNode myGraph n1
AddNode myGraph n2
AddNode myGraph n3
AddNode myGraph n4
AddNode myGraph n5

#Adds edges so that the graph is fully connected

AddEdge myGraph n1 n1 e11
AddEdge myGraph n1 n2 e12
AddEdge myGraph n1 n3 e13
AddEdge myGraph n1 n4 e14
AddEdge myGraph n1 n5 e15

AddEdge myGraph n2 n1 e21
AddEdge myGraph n2 n2 e22
AddEdge myGraph n2 n3 e23
AddEdge myGraph n2 n4 e24
AddEdge myGraph n2 n5 e25

AddEdge myGraph n3 n1 e31
AddEdge myGraph n3 n2 e32
AddEdge myGraph n3 n3 e33
AddEdge myGraph n3 n4 e34
AddEdge myGraph n3 n5 e35

AddEdge myGraph n4 n1 e41
AddEdge myGraph n4 n2 e42
AddEdge myGraph n4 n3 e43
AddEdge myGraph n4 n4 e44
AddEdge myGraph n4 n5 e45

AddEdge myGraph n5 n1 e51
AddEdge myGraph n5 n2 e52
AddEdge myGraph n5 n3 e53
AddEdge myGraph n5 n4 e54
AddEdge myGraph n5 n5 e55

#Lists the nodes in the graph
ListNodes myGraph
