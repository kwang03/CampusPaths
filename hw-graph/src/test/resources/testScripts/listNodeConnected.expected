#Tests the functionality of listing nodes in a connected graph

#Creates graph
created graph myGraph

#Adds some nodes to the graph
added node n1 to myGraph
added node n2 to myGraph
added node n3 to myGraph
added node n4 to myGraph
added node n5 to myGraph

#Adds edges so that the graph is fully connected

added edge e11 from n1 to n1 in myGraph
added edge e12 from n1 to n2 in myGraph
added edge e13 from n1 to n3 in myGraph
added edge e14 from n1 to n4 in myGraph
added edge e15 from n1 to n5 in myGraph

added edge e21 from n2 to n1 in myGraph
added edge e22 from n2 to n2 in myGraph
added edge e23 from n2 to n3 in myGraph
added edge e24 from n2 to n4 in myGraph
added edge e25 from n2 to n5 in myGraph

added edge e31 from n3 to n1 in myGraph
added edge e32 from n3 to n2 in myGraph
added edge e33 from n3 to n3 in myGraph
added edge e34 from n3 to n4 in myGraph
added edge e35 from n3 to n5 in myGraph

added edge e41 from n4 to n1 in myGraph
added edge e42 from n4 to n2 in myGraph
added edge e43 from n4 to n3 in myGraph
added edge e44 from n4 to n4 in myGraph
added edge e45 from n4 to n5 in myGraph

added edge e51 from n5 to n1 in myGraph
added edge e52 from n5 to n2 in myGraph
added edge e53 from n5 to n3 in myGraph
added edge e54 from n5 to n4 in myGraph
added edge e55 from n5 to n5 in myGraph

#Lists the nodes in the graph
myGraph contains: n1 n2 n3 n4 n5
